carp.common/src/commonMain/kotlin/dk/cachet/carp/common/application/users/AssignedTo.kt

```diff
@@ -0,0 +1,23 @@
+package dk.cachet.carp.common.application.users
+
+import kotlinx.serialization.Serializable
+
+
+/**
+ * Determines which participant roles to assign to something.
+ */
+@Serializable
+sealed class AssignedTo
+{
+    /**
+     * Assign this to anyone in the study protocol.
+     */
+    @Serializable
+    object Anyone : AssignedTo()
```

Comment:
I wonder if this should be named `AssignedTo.All` since that is actually what it means as far as I can understand. Anybody could imply any _single_ one. Or we might need an additional type here since the semantics are potentially different between assigning to a single participant and all participants?